.prettierrc

```diff
@@ -1,3 +1,5 @@
 {
-  "singleQuote": true
+  "singleQuote": true,
+  "trailingComma": "none",
+  "printWidth": 120
```

Comment:
I know that is just a personal preference but i think that with our modern monitor 150 or even  more is a better settings. I

Reply:
No, it's not a personal preference. I've started with 180 like we have in other repo, but the code goes offscreen and one needs to scroll horizontally to see everything. I tried the following setup: VS Code with the file browser panel open, on the macBook. 120 was the most comfortable but I can check 150 too.

Reply:
if you could check it yes it will be nice :)

Reply:
Updated, there are only a couple cases in e2e where one word goes off screen, the rest of the code is fine for 150 print width